dla_cnn/desi/preprocess.py

```diff
@@ -92,3 +92,68 @@ def label_sightline(sightline, kernel, REST_RANGE, pos_sample_kernel_percent=0.3
     # classification is 1 / 0 / -1 for DLA/nonDLA/border
     # offsets_array is offset
     return classification, offsets_array, column_density
+
+def rebin(sightline, v = 20000):
+    """
+    Resample and rebin the input Sightline object's data to a constant dlambda/lambda dispersion.
+
+    Parameters
+    ----------
+    sightline: dla_cnn.data_model.Sightline
+    v: float
+    
+    Returns
+    -------
+    sightline: dla_cnn.data_model.Sightline
+    """
+    c = 2.9979246e8
+    dlambda = np.log(1+v/c)
+    wavelength = np.exp(np.log(10)*sightline.loglam)
+    central_wavelength = [item.central_wavelength for item in sightline.dlas]
```

Comment:
I don't see why we would use the DLAs to define
the wavelength range.  Let's not do that.

e.g. what if there were no DLA in the sightline?

Reply:
Sure, I ignored the special cases and simply did as the linetools. I will change it.

Reply:
I have changed it. Thanks!